flexget/plugins/clients/qbittorrent.py

```diff
@@ -44,6 +45,7 @@ class OutputQBitTorrent:
                     'verify_cert': {'type': 'boolean'},
                     'path': {'type': 'string'},
                     'label': {'type': 'string'},
+                    'tags': {'type': 'array'},
```

Comment:
Should probably specify that the tags themselves have to be strings here.
```suggestion
                    'tags': {'type': 'array', 'items': {'type': 'string'}},
```